qa/src/test/java/org/eclipse/kapua/service/StepData.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
```

Comment:
I never know what exactely to put in Copiright. I have a templating, I have copyright plugin, but still it is about content.

Why is there 2011 at all? I just reused from other file. Is it there as the year when the file was created.
When is the year changed? Should it be changed in all files every year? Probably not.
It is probably only when file is changed by content.

In this case that you mentioned, it is probably as you say, just 2017 and no 2011.